src/lib.rs

```diff
@@ -277,3 +277,10 @@ pub mod futures {
         AsyncReadBytesExt, AsyncWriteBytesExt, BigEndian, LittleEndian, NativeEndian, NetworkEndian,
     };
 }
+/// Prelude for convinience
+pub mod prelude {
+    #[cfg(features = "tokio-traits")]
+    pub use tokio::io_tokio::{AsyncReadBytesExt, AsyncWriteBytesExt, BigEndian, LittleEndian, NativeEndian, NetworkEndian};
```

Comment:
I think you'll have to use the `use Trait as _` trick for the `Ext` traits so that the compiler won't yell at you if both features are enabled. The various `*Endian` types should also only be included once, since they should be the same for `tokio` and `futures`.